rebasebot/bot.py

```diff
@@ -466,6 +474,60 @@ def _manual_rebase_pr_in_repo(repo: github3.github.repo.Repository) -> Optional[
     return None
 
 
+def _push_rebase_branch(gitwd, rebase) -> bool:
```

Comment:
Would be cool to have type annotations here. And on all newly introduced functions.